content/en/_TEMPLATE.md

```diff
@@ -5,11 +5,13 @@ category: concept
 ---
 
 ## What it is
-A Quick summary of the concept, what it is.
+Quick summary of the concept and what it is.
 
 ## Problem it addresses
-A few lines of the problem it's addressing.
+Define the problem it addresses. Ideally, don't even mention the term you are defining. 
 
 ## How it helps
-A few lines of how the thing solves the problem.
+Describe how the term addresses the problem described above.
 
+## Related terms
+Add related Glossary terms (if applicable)
```

Comment:
I suggest to insert empty lines before and after the header lines. 😊

- https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines
- https://arcticicestudio.github.io/styleguide-markdown/rules/headings.html#empty-lines-before-and-after

Reply:
Sure! I'll make those changes

Reply:
lgtm!
let me know if I have to work on any changes